ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/BlobSidecarGossipValidator.java

```diff
@@ -63,14 +65,17 @@ public static BlobSidecarGossipValidator create(
     final Optional<Integer> maybeMaxBlobsPerBlock = spec.getMaxBlobsPerBlock();
 
     final int validInfoSize = VALID_BLOCK_SET_SIZE * maybeMaxBlobsPerBlock.orElse(1);
+    // It's not fatal if we miss something and we don't need finalized data
+    final int validSignedBlockHeadersSize = spec.getGenesisSpec().getSlotsPerEpoch() * 3;
```

Comment:
I'd rather extract the 3 to a config constant

Reply:
yeah, and it could have a proper name, it's not a magic number, will do it